dll/win32/shell32/utils.cpp

```diff
@@ -99,3 +99,134 @@ SHFindComputer(LPCITEMIDLIST pidlRoot, LPCITEMIDLIST pidlSavedSearch)
 
     return SUCCEEDED(hr);
 }
+
+static VOID Int64ToStr(LONGLONG llValue, LPWSTR pszValue)
+{
+    WCHAR szBuff[40];
+    INT ich = 0;
+#if (WINVER >= 0x0600)
+    BOOL bMinus = (llValue < 0);
+
+    if (bMinus)
+        llValue = -llValue;
+#endif
+
+    do
+    {
+        szBuff[ich++] = (WCHAR)(L'0' + (llValue % 10));
+        llValue /= 10;
+    } while (llValue != 0);
+
+#if (WINVER >= 0x0600)
```

Comment:
```suggestion
#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
```

Reply:
Non, non. `_WIN32_WINNT` is hacked at here. See `CMakeLists.txt`.